### PR TITLE
tweaked padding in headline

### PR DESCRIPTION
**Pull request by jonsaq**
@stewsmith Just needed to tweak the padding on the headline for the Opticon homepage

### CODE REVIEW

File: website-guts/assets/css/pages/opticon-homepage.scss [hunk 1]
```diff
@@ -8,7 +8,8 @@ body.opticon-homepage {
   .page-title {
     font-size: 2.6em;
     background: $opticon-brand-blue;
-    padding: 5px 20px;
+    padding: 0px 20px;
```
Comment (stewsmith) [inline website-guts/assets/css/pages/opticon-homepage.scss]: @jonsaq Could you correct this?

Comment (stewsmith) [inline website-guts/assets/css/pages/opticon-homepage.scss]: Also can you do this in one line as `padding: 10px 20px 0;`

**Comment by jonsaq** [comment]
@stewsmith Corrected padding syntax

**Comment by stewsmith** [comment]
LGTM

**Pull request merged**